tools/deployment/package-helm/templates/service-account.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: "v1"
+kind: "ServiceAccount"
+metadata:
+  name: {{ include "clp.fullname" . }}-service-account
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)